Dockerfile.docs

```diff
@@ -1,4 +1,6 @@
 FROM squidfunk/mkdocs-material
+LABEL org.opencontainers.image.url="https://hub.docker.com/r/rancher/rke2-runtime"
+LABEL org.opencontainers.image.source="https://github.com/rancher/rke2"
```

Comment:
This Dockerfile is no longer used; we were just using it to build the docs website before it was moved out to a different repo - to labeling it as published as this image is incorrect.

Reply:
totally makes sense, thanks for the feedback !